api/admin/controller/catalog_services.py

```diff
@@ -37,6 +38,7 @@ def process_catalog_services(self):
 
     def process_get(self):
         services = self._get_integration_info(ExternalIntegration.CATALOG_GOAL, self.protocols)
+        self.update_protocol_settings()
```

Comment:
Added this here because the list of Storage services became stale and was never updated. So if a new Storage integration was added or if an existing Storage integration updated its name, it did not appear correctly in the dropdown menu.